src/utils/types.ts

```diff
@@ -12,7 +12,7 @@ export type LiteralUnion<T extends U, U = string> =
  *
  * For some strange reason this is not the same as `Function`.
  */
-export type Callable = (...args) => unknown;
+export type Callable = (...args: any[]) => unknown;
```

Comment:
This can be typed generically to avoid explicit `any`:

```suggestion
export type Callable = <T>(...args: T[]) => unknown;
```

Reply:
I'm getting this

<img width="800" alt="image" src="https://user-images.githubusercontent.com/7195563/218706286-8239065d-7a17-42e0-a468-5d98f1446284.png">

Please help me further
I suggest checkout the branch and try to handle it on your own with your knowledge

Reply:
You are too quick xD. I updated my previous comment.

Reply:
Okay I mark these as resolved

Reply:
Please read my suggestion again. Guess you only read the first line.

> Since we are only interested in whether a type is a function (callable), this type can be narrowed to `export type Callable = () => void;`